app/components/shared/choice_personal_statement_component.html.erb

```diff
@@ -0,0 +1,16 @@
+<section id="personal-statement-section" class="app-section govuk-!-width-two-thirds">
+  <h2 class="govuk-heading-m govuk-!-font-size-27" id="personal-statement"><%= t('personal_statement.title') %></h2>
+
+    <% if @editable %>
```

Comment:
Is this component being used only on provider interface for now, correct? If so we might do not need this if and the block inside. If is used on candidate interface or support, ignore this comment.

Reply:
Updated the component by removing `editable` from the signature and moving the component to the `ProviderInterface` namespace. Also added a simple spec